temporalio/common.py

```diff
@@ -35,6 +37,21 @@ class RetryPolicy:
     non_retryable_error_types: Optional[Iterable[str]] = None
     """List of error types that are not retryable."""
 
+    @staticmethod
```

Comment:
nit: could use a `classmethod` instead and instaniate `cls()` below instead of referencing the class by name

Reply:
I like being explicit on class instantiation here (and I have chosen to use `@staticmethod` throughout, though Google style guide wants `@classmethod`; I have documented the deviation in the README too)